examples/awsbraket.ipynb

```diff
@@ -101,23 +104,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "    qureg = eng.allocate_qureg(3)\n",
-    "    # entangle the qureg\n",
-    "    H | qureg[0]\n",
-    "    H | qureg[1]\n",
-    "    C(X) | (qureg[1], qureg[2])\n",
-    "    C(X) | (qureg[0], qureg[1])\n",
-    "    H | qureg[0]\n",
-    "    C(X) | (qureg[1], qureg[2])\n",
-    "    # measure; should be all-0 or all-1\n",
-    "    All(Measure) | qureg\n",
-    "    # run the circuit\n",
-    "    eng.flush()\n"
+    "# Allocate the required qubits\n",
+    "qureg = eng.allocate_qureg(3)\n",
+    "\n",
+    "# Create the circuit. In this example a quantum teleportation algorithms that teleports the first qubit to the third one.\n",
+    "H | qureg[0]\n",
+    "H | qureg[1]\n",
+    "C(X) | (qureg[1], qureg[2])\n",
+    "C(X) | (qureg[0], qureg[1])\n",
+    "H | qureg[0]\n",
+    "C(X) | (qureg[1], qureg[2])\n",
+    "\n",
+    "# At the end we measure the qubits to get the results; should be all-0 or all-1\n",
+    "All(Measure) | qureg\n",
+    "\n",
+    "# And run the circuit\n",
+    "eng.flush()\n"
    ]
   },
   {
    "source": [
-    "Get the probabilities"
+    "The backend provides the task And, an identifier that we can use to recover the status of the Task and results later on.\n",
```

Comment:
```suggestion
    "The backend will automatically create the task and generate a unique identifier that that can be used to recover the status of the task and results later on.\n",
```

Reply:
with an small addition:

The backend will automatically create the task and generate a unique identifier (the task Arn) that can be used to recover the status of the task and results later on.